unittests/test_cli.py

```diff
@@ -71,15 +71,16 @@ def setUp(self):
         self.prefix = tempfile.mkdtemp(dir='unittests')
         self.system = 'generic:login'
         self.checkpath = ['unittests/resources/hellocheck.py']
-        self.environs  = ['builtin-gcc']
-        self.local  = True
+        self.environs = ['builtin-gcc']
+        self.local = True
         self.action = 'run'
         self.more_options = []
         self.mode = None
 
         # Monkey patch logging configuration
         self.logfile = os.path.join(self.prefix, 'reframe.log')
-        settings._logging_config = {
+        self.settings = config.load_from_file("reframe/settings.py")
```

Comment:
Please use single quotes here.

Reply:
With your feature there is no more need for the monkey-patching of settings. We could provide a custom settings file just for the unit tests (I also want this in my refactoring of the runtime resources and the new style checks). Assigning `settings` to `self` has no effect, that's why the logging unit tests fail. Before, we had to do the monkey-patching, because we had no way of providing alternative settings for the unit tests.